src/deploy.go

```diff
@@ -99,6 +118,9 @@ func shouldCompress(file string) bool {
 	return true
 }
 
+/*
+* Upload file s3
```

Comment:
The only information you add is that it's to s3, and we probably already know that.